examples/08-Filtersolutions/filltersolutionsexample.py

```diff
@@ -0,0 +1,42 @@
+# # Design a Lumped Element Filter
```

Comment:
@ramin4667 Examples should keep the same syntax and way of working. Please check one of the available examples and try to adapt your script. It is important, because when we create a new release, we run all examples and we create the HTML. So if there is something wrong, the full release can fail. You can skip for now this file, and once the core code is integrated, we can add the example in a new PR (Pull request)

Reply:
Also, please keep the same file name standard if possible (01-Filter....)

Reply:
@Samuelopez-ansys The example syntax is updated.